services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -19,6 +19,7 @@ import {
   FormField,
   InputChangeEvent,
 } from "types";
+import { labelTextWithOptional } from "utils/other/styleAsOptional";
```

Comment:
Request: Change to export util the same as all other utils through the `utils` dir, then import with the others above on lines 9-12.

Reply:
_Note: Applies to all instances below._